website/content/platform/development/contributing/how-to/add_data_to_existing_endpoint.md

```diff
@@ -0,0 +1,860 @@
+---
+title: How To Add A Data Provider To An Existing Endpoint
+sidebar_position: 2
+description: This guide outlines the process for adding an endpoint to an existing data provider and router endpoint.
+keywords:
+- OpenBB Platform
+- Open source
+- Python interface
+- REST API
+- contribution
+- contributing
+- documentation
+- code
+- provider
+- data
+- endpoint
+- existing
+- OpenBB extensions
+- OpenBB provider
+- standard model
+- provider model
+- how to
+---
+
+import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
+
+<HeadTitle title="How To Add A Data Provider To An Existing Endpoint - Contributing - Development | OpenBB Platform Docs" />
+
+This page will walk through adding a data provider to an existing endpoint, using a standard model. At a high level, the process will look something like:
+
+- Catalogue the parameters and returned fields from the chosen data provider.
+- Find the existing standard model that is mapped to the router endpoint.
+- Identify any common parameters and fields to map.
+- Build the provider models and Fetcher class by inheriting from the standard models.
+- Map the new provider model to the router.
+- Rebuild the Python interface and static assets.
+- Add unit tests.
+- Add integration tests.
+- Submit a pull request.
+
+:::note
+
+Before getting started, get a few housekeeping items in order:
+
+- Clone the GitHub repo and navigate into the project's folder.
+  - If you have already done this, update your local branch:
+      - `git fetch`
+      - `git pull origin develop`
+- Install the OpenBB Platform in "editable" mode.
+  - `cd openbb_platform`
+  - `python dev_install.py -e`
+- Rebuild the Python interface and static assets.
+  - `import openbb`
+  - `openbb.build()`
+- Create a new local branch (pick a relevant name and use dashes for multiple words), always beginning with `feature/`.
+  - `git checkout -b feature/av-historical-eps`
+
+:::
+
+Let's get started!
+
+## Identify What's Being Added
+
+For this example, we will be adding a user-requested data set from AlphaVantage, historical EPS. There is an existing [GitHub issue](https://github.com/OpenBB-finance/OpenBBTerminal/issues/6104) that we will link to in the submitted pull request. By linking the issue, it will be closed automatically on merge.
```

Comment:
Is there a need to link to an issue for this example? Is it out of scope of what the example should show?